spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/kms/GcpKmsAutoConfiguration.java

```diff
@@ -65,13 +70,17 @@ GcpProjectIdProvider getGcpProjectIdProvider() {
   @ConditionalOnMissingBean
   public KeyManagementServiceClient keyManagementClient(CredentialsProvider googleCredentials)
       throws IOException {
-    KeyManagementServiceSettings settings =
+    KeyManagementServiceSettings.Builder settings =
```

Comment:
nit: Rename the variable as well since the type is changed. All the references in this method also need to be updated.
```suggestion
    KeyManagementServiceSettings.Builder settingsBuilder =
```

Reply:
Done, thank you.